v3/plugins/ms-vscode/typescript/1.49.3/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: ms-vscode
+name: typescript
+version: 1.49.3
+type: VS Code extension
+displayName: Typescript
+title: Typescript language features
+description: Typescript language features
+icon: https://raw.githubusercontent.com/microsoft/vscode/1.49.3/extensions/typescript-language-features/icon.png
+repository: https://github.com/Microsoft/vscode
+category: Language
+firstPublicationDate: '2020-10-22'
+spec:
+  containers:
+  - image: "quay.io/eclipse/che-sidecar-node:12-026416c"
```

Comment:
Let's use the new [integrated sidecar images](https://quay.io/repository/eclipse/che-plugin-sidecar?tag=latest&tab=tags), so `quay.io/eclipse/che-plugin-sidecar:node-c939ba4`